network-providers/basic/resources.md

```diff
@@ -0,0 +1,211 @@
+## NetworkConfig
+
+```
+dto NetworkConfig:
+    raw: any;
```

Comment:
`raw: any;` - this means that all resources returned by network providers present not only explicit, typed fields, but also hold the original payload of the HTTP response.